backend/.env.template

```diff
@@ -17,6 +17,7 @@ PRIVATE_KEY_PASSPHRASE="a7dsf78sadg87ad87sfagsadg78"
 SENTRY_DSN_BACKEND=
 COMMIT=
 PUBLIC_REGISTRATION=false
+INVITE_REGISTRATION=true
```

Comment:
Do we really want this set to true in the template? I think it is better to set it to false